conanfile.py

```diff
@@ -165,6 +173,9 @@ def _disableQtOptions(disableFlag, options):
     def requirements(self):
         # TODO: will no longer be needed after merging https://github.com/conan-io/conan-center-index/pull/13399
         self.requires("libpng/[~1.6.38]", override=True) # freetype / Qt
+        if self.settings.os == "Windows":
+            self.requires("zlib/[~1.2.13]", override=True) # minizip / Qt
+            self.requires("libiconv/[~1.17]", override=True) # ffmpeg / sdl
```

Comment:
not sure, but aren't those versions already used by default? or something still requires older/newer?

Reply:
@kambala-decapitator because there is conflicts - minizip wants 1.2.12 and Qt wants 1.2.13,

for second - sdl wants 1.17 and ffmpeg wants 1.16

Reply:
clear, thanks. Then please move these 2 lines to `else` branch of `use Apple system libraries instead of external ones`

Reply:
done